wiki/en/Server-Install.md

```diff
@@ -0,0 +1,97 @@
+
+### Installation
+
+1. Download the [latest headless .deb file]({{ site.download_root_link }}{{ site.download_file_names.deb-headless }})
+1. Update apt to make sure you have a current list of standard packages: `sudo apt update`
+1. Install the Jamulus package: `sudo apt install ./{{ site.download_file_names.deb-headless }}`
+1. Enable the headless Server process via systemd: `sudo systemctl enable jamulus-headless`
+1. Add your desired [command line options](Running-a-Server#command-line-options) to the `ExecStart` line in the systemd service file in `/lib/systemd/system/jamulus-headless.service` (By default you will be running an Unregistered Server).
+1. Reload the systemd files `sudo systemctl daemon-reload` and restart the headless Server: `sudo systemctl restart jamulus-headless`
+1. Check all is well with `systemctl status jamulus-headless` (hit `q` to get back to the command prompt).
+
+**Note**: You can upgrade your server by repeating the steps mentioned above.
+
+### Configuration
+
+You can control Jamulus with the `systemctl` command. For example, to stop the Server cleanly:
+
+`sudo systemctl stop jamulus-headless`
+
+### Running an Unregistered Server
+
+It is highly recommended to test your Server on a **Public Directory** first so as to narrow down any subsequent problems in unregistered mode.
+
+### Port forwarding
+
+If you are running a Server on your home network, people from outside will not be able to see things inside it. So if you want to operate a Jamulus Server, you need to open a hole in your router's firewall to let Jamulus Clients connect to it.
+
+**The default port for the current version of Jamulus is UDP (not TCP) port 22124.**
+
+The exact setup of port forwarding differs for every router. For help see [portforward.com](https://portforward.com).
+
+Once you have your router set up, you can get your external (WAN) IP address e.g. by [using Google](https://www.google.com/search?q=what+is+my+ip). Give this address to your friends so they can connect to your Server (_but see also the note on dynamic DNS below_). You yourself should connect using the local network (LAN) address of the machine the Server is running on. If you are running a Client on the same machine as your Server, that would be `localhost` or `127.0.0.1`.
+
+## Dynamic DNS and why you will probably need it
+
+Home internet connections ususally change their IP address after a short period (hours, days or weeks). To allow people to connect to you, you probably want set up a dynamic DNS address for getting a static (sub-)domain people can connect to. You can do this on [the machine](https://www.online-tech-tips.com/computer-tips/ddns-dynamic-dns-service/) that you are running the Jamulus Server on, or preferably your router [may support it](https://www.noip.com/support/knowledgebase/how-to-configure-ddns-in-router/).
```

Comment:
> Home internet connections **ususally** change

"usually"

Reply:
> you probably want set up a dynamic DNS address for getting a static (sub-)domain people can connect to.

Err... this is way too vague.  And weirdly worded.

Reply:
Given the section heading, "Dynamic DNS and why you will probably need it", I'd leave it at explaining why - not try to explain what to do about it if it's going to end up with this mess.  Trying to explain how to set up DDNS really is beyond the scope of installing a Jamulus Server.

Reply:
```suggestion
Home internet connections usually change their IP address after a short period (hours, days or weeks). To allow people to connect to you, you probably want set up a dynamic DNS address for getting a static (sub-)domain people can connect to. You can do this on [the machine](https://www.online-tech-tips.com/computer-tips/ddns-dynamic-dns-service/) that you are running the Jamulus Server on, or preferably your router [may support it](https://www.noip.com/support/knowledgebase/how-to-configure-ddns-in-router/).
```

Reply:
If we are looking to trim things down, I think we might want to consider removing this information entirely (or perhaps moving to T&T) on the grounds that it's not core to the operation of Jamulus and tempts us to start teaching people about stuff they can better find on Google.